q2_composition/tests/test_ancombc.py

```diff
@@ -50,9 +55,26 @@ def test_ids_in_table_not_in_md(self):
         with self.assertRaisesRegex(KeyError, 'Not all samples present within'
                                     ' the table were found in the associated'
                                     ' metadata file.*L6S68'):
-            ancombc(table=self.table, metadata=self.missing_md,
+            ancombc(table=self.table, metadata=self.md_missing_md,
                     formula='bodysite')
 
+    # TODO: IDs with Es that look like scientific notation 004e002
+    def test_ids_in_table_with_es(self):
```

Comment:
I just copied a function below as a placeholder. This is checking for columns, while we are worried about sampleIDs